site/content/en/docs/_index.md

```diff
@@ -20,6 +20,7 @@ minikube quickly sets up a local Kubernetes cluster on macOS, Linux, and Windows
 * Deploy as a VM, a container, or on bare-metal
 * Multiple container runtimes (CRI-O, containerd, docker)
 * Direct API endpoint for blazing fast [image load and build]({{< ref "/docs/handbook/pushing.md" >}})
+* Support for creating and testing [network policy]({{< ref "docs/handbook/network_policy.md" >}}) (both Kubernetes and Calico network policy)
```

Comment:
Thank you for this contribution, 
for sake of keeping the first page short and sweet , how about adding this part of the Advanced features bellow


* Advanced features such as [LoadBalancer]({{< ref "/docs/handbook/accessing.md#loadbalancer-access" >}}), filesystem mounts, and FeatureGates and Networkpolicy

Reply:
Thanks @medyagh, I agree! I have made that change.